TarSCM/scm/git.py

```diff
@@ -251,7 +251,7 @@ def prepare_working_copy(self):
         # between multiple services
         org_clone_dir  = self.clone_dir
         self.clone_dir = self.repodir
-        command = ['git', 'clone', self.repocachedir, self.url, self.clone_dir]
+        command = ['git', 'clone', '--no-checkout']
```

Comment:
If you do a "osc build" on your local checkout, the git repo is cached. You can do your changes in this local cache and rerun "obs build". The local changes are used for these builds. When you are done, you simply commit those changes to your git repo and do a "osc service rr" and the obs server takes your changes from your git repo, builds a package and publishes the new package.

This would be broken if we would do '--no-checkout' in general.